crates/vault-registry/src/default_weights.rs

```diff
@@ -51,4 +55,16 @@ impl crate::WeightInfo for () {
             .saturating_add(DbWeight::get().reads(1 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
+    fn adjust_collateral_ceiling() -> Weight {
+        6_788_000_u64.saturating_add(DbWeight::get().writes(1 as Weight))
```

Comment:
Note: I guessed these weights based on similar write-only functions in other crates. Let me know if this needs to be adjusted or there's a better way to calculate them

Reply:
The proper way is to add a benchmark in `benchmarking.rs`, which shouldn't be hard for basic setters like this. The actual weight is not so important as this function will only very rarely be called, but it does make updating the weights easier (the benchmark outputs code that we can just paste in default_weights.rs - if one of the functions does not have a benchmark you need to manually copy-paste the old one). If you haven't worked with benchmarks before, and if you have spare time, I would suggest you try to add the benchmark - if nothing else, it's good to have done once